frontend/public/locales/zh-CN/common.json

```diff
@@ -0,0 +1,219 @@
+    "team_count_select_elimination_placeholder":"2，4，8等",
+    "team_count_input_round_robin_label": "从上一阶段晋级的队伍数量",
+    "none": "无",
+    "at_least_two_team_validation": "需要至少两个队伍",
+    "stage_type_select_label": "阶段类型",
+    "round_robin_label": "循环赛",
+    "single_elimination_label": "单淘汰赛",
+    "swiss_label": "瑞士制",
+    "create_stage_item_button": "创建阶段项目",
+    "add_stage_item_modal_title": "添加阶段项目",
+    "remove_match_button": "移除比赛",
+    "negative_score_validation": "分数不能为负",
+    "negative_match_duration_validation": "比赛持续时间不能为负",
+    "negative_match_margin_validation": "比赛间隔时间不能为负",
+    "edit_match_modal_title": "编辑比赛",
+    "score_of_label": "得分",
+    "custom_match_duration_label": "自定义比赛持续时间",
+    "customize_checkbox_label": "自定义",
+    "custom_match_margin_label": "自定义比赛间隔时间",
+    "minutes": "分钟",
+    "at_least_one_player_validation": "输入至少一个玩家",
+    "active_players_checkbox_label": "这些玩家是活跃的",
+    "active_player_checkbox_label": "这个玩家是活跃的",
+    "player_name_input_placeholder": "最佳玩家",
+    "save_players_button": "保存玩家",
+    "single_player_title": "单个玩家",
+    "multiple_players_title": "多个玩家",
+    "create_player_modal_title": "创建玩家",
+    "add_player_button": "添加玩家",
+    "edit_player": "编辑玩家",
+    "delete_round_button": "删除轮次",
+    "edit_round": "编辑轮次",
+    "round_name_input_placeholder": "最佳轮次",
+    "active_round_checkbox_label": "这个轮次是活跃的",
+    "draft_round_checkbox_label":"这个轮次是草稿轮次",
+    "home_title": "主页",
+    "home_spotlight_description": "前往主页",
+    "clubs_spotlight_description": "查看、添加或删除俱乐部",
+    "user_settings_title": "用户设置",
+    "user_settings_spotlight_description": "更改名称、电子邮件、密码等",
+    "planning_title": "规划",
+    "planning_spotlight_description": "更改比赛规划",
+    "teams_title": "队伍",
+    "teams_spotlight_description": "查看、添加或删除队伍",
+    "players_title": "玩家",
+    "players_spotlight_description": "查看、添加或删除玩家",
+    "stage_title": "阶段",
+    "stage_spotlight_description": "更改锦标赛的布局",
+    "court_title": "场地",
+    "court_spotlight_description": "查看、添加或删除场地",
+    "tournament_setting_title": "锦标赛设置",
+    "tournament_setting_spotlight_description": "更改锦标赛的设置",
+    "nothing_found_placeholder": "未找到任何内容...",
+    "search_placeholder": "搜索...",
+    "at_least_one_team_validation": "输入至少一个队伍",
+    "active_teams_checkbox_label": "这些队伍是活跃的",
+    "team_name_input_placeholder": "最佳队伍",
+    "active_team_checkbox_label": "这个队伍是活跃的",
+    "team_member_select_placeholder": "选择你喜欢的所有成员",
+    "team_member_select_title": "队伍成员",
+    "single_team": "单个队伍",
+    "multiple_teams": "多个队伍",
+    "add_team_button": "添加队伍",
+    "edit_team_title": "编辑队伍",
+    "club_select_label": "俱乐部",
+    "now_button": "现在",
+    "logout_title": "登出",
+    "user_title": "用户",
+    "more_title": "更多",
+    "website_title": "网站",
+    "github_title": "Github",
+    "api_docs_title": "API 文档",
+    "elo_input_label": "最大 ELO 差值",
+    "only_recommended_input_group_label": "只显示比赛次数较少的队伍",
```

Comment:
The indentation here is slightly off, could you remove the two spaces at the start of each line?

Reply:
Fixed